### PR TITLE
janus 1.0.0 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
janus 1.0.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-3928]
- dev_url: https://github.com/aio-libs/janus/tree/v1.0.0
- conda-forge: https://github.com/conda-forge/janus-feedstock/blob/main/recipe/meta.yaml
- pypi: https://pypi.org/project/janus
- pypi inspector: https://inspector.pypi.io/project/janus/1.0.0

### Explanation of changes:

- recipe, add abs, add upstream tests, no docs link found

[PKG-3928]: https://anaconda.atlassian.net/browse/PKG-3928?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by boldorider4 — COMMENTED** [review]
isn't this a docs page for this package?
https://ccqc.github.io/janus/_build/html/source/intro.html

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,26 +11,44 @@ source:
...
-  noarch: python
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
+    - python
     - pip
-    - python >=3.7
+    - setuptools
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: setuptools>=51 ? , see  https://github.com/aio-libs/janus/blob/0783f9b7a9bb7e1c095e93ebb4aad4f1e219f512/pyproject.toml#L2C14-L2C28

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,26 +11,44 @@ source:
...
+    - python
     - typing_extensions >=3.7.4.3
 
 test:
   imports:
     - janus
+  requires:
+    - pip
+    - pytest
+    - pytest-asyncio
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: pytest-asyncio>= 0.16.0 ? , see https://github.com/aio-libs/janus/blob/0783f9b7a9bb7e1c095e93ebb4aad4f1e219f512/requirements-dev.txt#L11C1-L11C23

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,26 +11,44 @@ source:
...
-    - python >=3.7
+    - python
     - typing_extensions >=3.7.4.3
 
 test:
   imports:
     - janus
+  requires:
+    - pip
+    - pytest
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: pytest >=6.2.5 ? see https://github.com/aio-libs/janus/blob/0783f9b7a9bb7e1c095e93ebb4aad4f1e219f512/requirements-dev.txt#L10

**Comment by lorepirri** [comment]
> isn't this a docs page for this package? https://ccqc.github.io/janus/_build/html/source/intro.html

@boldorider4 that one seems to be an open-source python library for the implementation and application of adaptive QM/MM methods. https://ccqc.github.io/janus/_build/html/source/intro.html pointing to another repo [repo](https://github.com/CCQC/janus/) (see https://ccqc.github.io/janus/_build/html/source/setting-up-janus.html)

the package on pypi points to this one https://github.com/aio-libs/janus and it's the same on conda-forge

File: recipe/meta.yaml (hunk 3 above)
Comment (lorepirri) [reply recipe/meta.yaml]:
```suggestion
    - pytest >=6.2.5
```

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml]:
```suggestion
    - pytest-asyncio >=0.16.0
```

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]:
```suggestion
    - setuptools >=51
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -11,26 +11,44 @@ source:
...
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
+    - python
     - pip
-    - python >=3.7
+    - setuptools
+    - wheel
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
    - wheel >=0.36
```

**Comment by lorepirri** [comment]
thanks for reviewing this one @boldorider4 and @M-Waszkiewicz-Anaconda, I've addressed all the concerns.

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by boldorider4 — APPROVED** [review]

**Pull request merged**